.pre-commit-config.yaml

```
@@ -79,13 +79,13 @@ repos:
     - tomli
 
 - repo: https://github.com/crate-ci/typos
-  rev: v1.23.6
+  rev: typos-dict-v0.11.27
   hooks:
   - id: typos
     name: typos (add false positives to _typos.toml)
 
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.5.6
+  rev: v0.6.3
```

```suggestion
  rev: v0.5.7
```
Upgrade ruff, but only to the most recent version where it didn't check Jupyter notebooks.